commands/gopher.go

```diff
@@ -47,40 +51,38 @@ func Gopher(ds *discordgo.Session, dm *discordgo.Message, ctx *mux.Context) {
 	if err != nil {
 
 		// Try fetching via REST API
-		g, err = ds.Guild(dm.ChannelID)
+		g, err = ds.Guild(ic.ChannelID)
 		if err != nil {
-			lit.Error("getting guild, %s", err)
-			ds.ChannelMessageSend(dm.ChannelID, `Looks like all the Gophers are sleeping right now`)
-			return
+			lit.Error("gopher: getting guild: %s", err)
+			return nil, gopherErr
 		}
 	}
 
 	// Look for the message sender in that guild's current voice states.
 	for _, vs := range g.VoiceStates {
-
-		if vs.UserID == dm.Author.ID {
-			err = playSound(ds, g.ID, vs.ChannelID)
-			if err != nil {
-				lit.Error("play sound, %s", err)
-				ds.ChannelMessageSend(dm.ChannelID, `Looks like all the Gophers are sleeping right now`)
-			}
-			return
+		if vs.UserID == ic.Message.Author.ID {
+			// Call in goroutine to allow functon to return response
+			go func() {
+				err = playSound(ds, g.ID, vs.ChannelID)
+				if err != nil {
+					lit.Error("gopher: playing sound: %s", err)
```

Comment:
Lost the `Looks like all the Gophers are sleeping right now` message. Maybe make it a followup message.